src/data/markdown/docs/02 javascript api/08 k6-http/10-batch- requests -.md

```diff
@@ -111,7 +100,33 @@ export default function () {
 
 </CodeGroup>
 
-_Note that the requests in the example above may happen in any order, or simultaneously. When running requests in batches, there is no guarantee that e.g. req1 will happen before req2 or req3_
+_Note that the requests in the example above may happen in any order, or simultaneously. When running requests in batches, there is no guarantee that e.g. req1 will happen before req2 or req3_.
+
+### Example with GET shorthand
+
+If a requests' item is passed as a `string` then they will be automatically parsed as a `GET` request. 
```

Comment:
Again, we should specify whether this works on only arrays, or on arrays and objects.